scripts/digest_inputs

```diff
@@ -32,6 +32,19 @@ DEV_TAGS="${DEV_TAGS:-}"
 DOCKERFILE="${DOCKERFILE:-Dockerfile}"
 TAGS="${TAGS:-}"
 REDHAT_TAG="${REDHAT_TAG:-}"
+PUSH_AUTO_DEV_TAGS="${PUSH_AUTO_DEV_TAGS:=false}"
+
+# Strip version to MAJOR.MINOR
+MINOR="${VERSION%.*}"
```

Comment:
Any chance ``$VERSION`` could have more than 3 segments or a suffix?

Reply:
It should work for cases like `1.0.0-dev`, but come to think of it, there are some edgecases like `tfc` that uses `release/*.*.*-rc.x` or boundary hcp that uses `1.0.0+hcp.int` - Ill change the logic to parse only first two segments.

Reply:
(the metadata should technically never be passed in, but ill add this for safety)